__tests__/app.test.js

```diff
@@ -36,4 +36,36 @@ describe('GET /api', () => {
             expect(res.body).toEqual(endpoints)
         })
     })
+})
+
+describe('GET /api/reviews/:review_id', () => {
+    test('GET - STATUS: 200 - responds with review object of corresponding review_id endpoint', () => {
+        return request(app).get('/api/reviews/1')
+        .expect(200)
+        .then((res) => {
+            expect(typeof res.body).toBe('object');
+            expect(res.body.review.title).toBe('Agricola');
```

Comment:
An expect statement for the article_id of this object is arguably the most important one here!